packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -298,6 +306,22 @@ struct WindowData {
             self->m_useDirectDebugger = IsDlgButtonChecked(hwnd, IDC_DIRECTDEBUGGER) == BST_CHECKED;
             self->m_breakOnNextLine = IsDlgButtonChecked(hwnd, IDC_BREAKONNEXTLINE) == BST_CHECKED;
 
+            LPWSTR buffer = new WCHAR[BUFFER_LENGTH];
```

Comment:
>BUFFER_LENGTH [](start = 38, length = 13)

actually it'd be even better if there is no such #define. Instead, do

wchar_t buffer[6] = {};

then whenever you need the number of elements, do ARRAYSIZE(buffer)